python/tests/ecl_tests/test_sum.py

```diff
@@ -393,3 +377,22 @@ def test_resample(self):
         time_vector_resample = case2.alloc_time_vector(False)
         first_diff = time_vector_resample.first_neq( time_vector)
         self.assertEqual( time_vector_resample, time_vector)
+
+
+    def test_restart_abs_path(self):
+        with TestAreaContext("restart_test"):
+           history =  create_case(case = "HISTORY")
+           history.fwrite()
+           with scope():
+               restart_case, _ = os.path.splitext(os.path.abspath("HISTORY.SMSPEC"))
+               with pushd("subdir"):
+                   prediction = create_case( case = "PREDICTION", restart_case = restart_case, data_start = history.end_date)
+                   prediction.fwrite()
+
```

Comment:
I think you need to explain what you are thinking here.

Reply:
This was to reproduce / test a reasonably contrived bug. Have added documentation describing the situation which used to fail.